modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -335,7 +342,37 @@ ifdef::openshift-origin[]
 <15> Provide the `imageContentSources` section from the output of the command to mirror the repository.
 endif::openshift-origin[]
 endif::restricted[]
-
+ifndef::ibm-z,ibm-z-kvm,ibm-power,agnostic,rhv,openshift-origin[]
+ifndef::restricted[]
+<15> Activate a predefined set of cluster capabilities.
+<16> Add the additional cluster capabilities you want to enable here.
```

Comment:
Should we tell the supported values? Or should we route users to find the supported values?

Reply:
We have more information earlier in the page in the "installation configuration parameters" table, which also makes reference to the "Cluster capabilities" page with all of the details. I think it would be OK not to list everything in the annotations.